README.md

```diff
@@ -100,9 +100,7 @@ Complete installation requirements can be found in the documentation under <a hr
 
 **Node:**
 
-Please note at this time Node v14 **is not** supported
-
-- NodeJS >= 10.16 <13
+- NodeJS >= 10.16 <14
```

Comment:
Doesn't this 'translate' to Node.js must be greater then or equal to `v10.16` and less then `v14`?

Does Strapi still support `^10.16`?

```suggestion
- NodeJS 12.x || 14.x
```

Reply:
I'll leave that up to @alexandrebodin as to what we support (I agree we should probably drop v10, and not support v13)

Reply:
We still support node 10 for now